pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -212,6 +212,23 @@ def create_returnn_layer_dict(self, *inputs: Tensor) -> Dict[str, Any]:
       "from": [self._get_input_layer_name(x) for x in inputs]}
 
 
+class Tile(Module):
+  """
+  Wraps RETURNN TileLayer.
+  """
+  is_original_torch_module = False
+
+  def __init__(self, multiples: Dict[str, int]):
```

Comment:
A PyTorch module should not mix with RETURNN logic here. The keys are RETURNN axes. That doesn't make sense.
The call to `get_returnn_axis_description` should actually be in `create_returnn_layer_dict` below.